README.md

```diff
@@ -1,2 +1,36 @@
 # gh-dependabot
-A GH CLI extension for Dependabot
+A GH CLI extension for using Dependabot
+
+## Installation
+
+```bash
+gh extension install https://github.com/therealkujo/gh-dependabot
```

Comment:
Shorter and sweeter 🍬 

```suggestion
gh extension install therealkujo/gh-dependabot
```

Reply:
I see `requirements.txt` which I assume requires installing system dependencies or setting up a `venv`.

Should that be incorporated here?  Any concerns about managing external dependencies with the extension?

Reply:
Ahh great point! Completely forgot to have a step to install the one dependency. I don't think there will be an issue with managing that dependency as it just helps to manage the interface for the script and doesn't affect the functionality of the code itself.